openspec/changes/project-02-plan-root-command-fix/tasks.md

```
@@ -0,0 +1,4 @@
+- [x] 1. Update spec delta for project command surface to require `plan` command exposure from specfact-project bundle.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)